packages/web-ui/src/ds/molecules/FocusFooter/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react'
+
+export default function FocusFooter({ content }: { content: ReactNode }) {
```

Comment:
This is super specific for the signup. Doesn't look like something part of design system.